pkg/capabilities/capabilities.go

```diff
@@ -40,6 +40,14 @@ func Initialize(c Capabilities) {
 	})
 }
 
+// Setup the capability set.  It wraps Initialize for improving usibility.
+func Setup(allowPrivileged bool, hostNetworkSources []string) {
```

Comment:
I'm not a fan of this factoring, but it's not your problem.  @erictune @vmarmol things like `hostNetworkSources` is pretty kubelet specific, why is this outside pkg/kubelet?